lib/Backend/LowerMDSharedSimd128.cpp

```diff
@@ -1327,7 +1336,11 @@ IR::Instr* LowererMD::Simd128LowerShift(IR::Instr *instr)
         IR::RegOpnd * shamtReg = IR::RegOpnd::New(TyInt8, m_func);
         shamtReg->SetReg(LowererMDArch::GetRegShiftCount());
         IR::RegOpnd * tmp = IR::RegOpnd::New(TyInt8, m_func);
-
+#ifdef _M_IX86
```

Comment:
Yes, so I see. I wonder if the right fix isn't to require that a def and uses of a RegOpnd all have the same type, so that it's up to the lowered code gen to insert moves to guarantee this. We can look into that this week, but for now this emergency fix will do.